packages/polar/src/lib/deploy/contract.ts

```diff
@@ -287,15 +288,17 @@ export class Contract {
       });
     }
     // Send execute msg to the contract
-    const signingClient = await getSigningClient(this.env.network, accountVal);
+    const signingClient = await getSigningClient(this.env.network, accountVal, this.customFees);
 
     const msgData: { [key: string]: object } = {}; // eslint-disable-line @typescript-eslint/ban-types
     msgData[methodName] = callArgs;
     console.log(this.contractAddress, msgData);
     // Send the same handleMsg to increment multiple times
     return await signingClient.execute(
       this.contractAddress,
-      msgData
+      msgData,
+      undefined,
```

Comment:
what is the use of this variable ? do we need undefined here ?

Reply:
I needed to skip this optional parameter `memo`, passing it as `undefined` is the only way I found.